xtask/tests/tidy.rs

```diff
@@ -82,7 +84,10 @@ Zlib
         .collect::<Vec<_>>();
     licenses.sort();
     licenses.dedup();
-    assert_eq!(licenses, expected);
```

Comment:
This is intentionally exact, rather than subset, comparison

Reply:
Reverted.